flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "skip-appstream-check": true
```

Comment:
Appdata is useful also from CLI for `flatpak search`.

Reply:
I think we do actually ship an appdata file -- org.freedesktop.fwupd.metainfo.xml -- no OARS rating as I'm not sure it makes sense at all.